README.md

```diff
@@ -128,8 +130,30 @@ Alternatively, you can use the [CLI](#kogito-cli) to deploy your services:
 $ kogito deploy-service example-quarkus https://github.com/kiegroup/kogito-examples/ --context-dir=drools-quarkus-example
 ```
 
+### Clean up a Kogito Service deployment
+
+```bash
+$ kogito delete-service example-quarkus
+```
+
+### Native X JVM Builds
+
+By default, the Kogito Services will be built with traditional `java` compilers to speed up the time and save resources. This means that the final generated artifact will be a [uber jar](https://stackoverflow.com/questions/11947037/what-is-an-uber-jar) with the chosen runtime (default to Quarkus).
```

Comment:
it is not a uber jar, the build generates a jar with no dependencies on it, the dependencies will be copied to the runtime image (lib/ directory).

Reply:
Well noted. Updated!